parser/src/cfg/mod.rs

```diff
@@ -2646,31 +2646,39 @@ fn parse_layers(s: &mut ParsedState, mapped_keys: &mut MappedKeys) -> Result<Int
                     let input = &triplet[0];
                     let mapstr = &triplet[1];
                     let action = &triplet[2];
-                    let input_key = input
-                        .atom(s.vars())
-                        .and_then(str_to_oscode)
-                        .ok_or_else(|| anyhow_expr!(input, "input must be a key name"))?;
-                    mapped_keys.insert(input_key);
-                    if !layer_mapped_keys.insert(input_key) {
-                        bail_expr!(input, "input key must not be repeated within a layer")
-                    }
-                    let mapstrs = ["=", ":", "->", ">>", "maps-to", "→", "🞂"];
-                    mapstr
-                        .atom(s.vars())
-                        .and_then(|s| match mapstrs.contains(&s) {
-                            true => Some(()),
-                            false => None,
-                        })
-                        .ok_or_else(|| {
-                            anyhow_expr!(
-                                mapstr,
-                                "map string must be one of the following strings:\n\
+                    if input.atom(s.vars()).is_some_and(|x| x == "_") {
+                        let action = parse_action(action, s)?;
+                        for (i, _) in s.mapping_order.iter().enumerate() {
```

Comment:
Should we be checking to only overwrite `Trans` here?

Reply:
Yes, we should, at first I wasn't sure we should, now that I thought of it, we can rely on this check, because it doesn't make sense to explicitly map a key to `Trans`, since it is mapped to `Trans` by default when the layer array is created.

let me know if I didn't do it quite right in the upcoming commit.